### ISSUE TITLE
Training/Test Data & Computating Resources

### ISSUE THREAD

**Issue by GloryyrolG** [CSJasper/HMTV#4]
Hi @CSJasper @YangChangHee et al.,

thx for ur great work. may i ask what the training data is used & how long the training takes (e.g., x days on x gpus)? looking fwd to ur reply.

best,

**Comment by CSJasper** [comment]
Hello, thank you for your interest in our research. Our methodology enhances performance without additional training. As a result, no further training is required, and we utilize pre-trained models to generate 3D motion, which is then projected to improve the performance of existing text-to-video (T2V) models. If you have any further questions, please feel free to ask anytime.

**Comment by GloryyrolG** [comment]
Hi @CSJasper ,

thx for ur quick reply. im still not quite sure about how to get a pose-guided t2v (like cogvideo, t2v-0; while fyp I understood comes with pose conditions) model in a training-free manner? thx & rgds,

**Comment by GloryyrolG** [comment]
Hi @CSJasper ,

another question, may i confirm that if the evaluation is performed on,

![image](https://github.com/user-attachments/assets/275f937c-b0e1-472e-8faf-6e80f8ea4c77)

![image](https://github.com/user-attachments/assets/fd72ebd7-3137-4e32-a0e0-45fcd6598078)

https://github.com/CSJasper/HMTV/blob/3a2e376df0257f13874a47c1ae276a713113ae3e/evaluation/classification_accuracy.py#L131

**Comment by CSJasper** [comment]
Thank you for asking a great question. Our methodology involves automatically generating pose guidance using a T2M model when using models that rely on pose guidance, such as T2V-Zero or FYP. This allows us to create videos featuring people with better quality. Therefore, our approach can only be applied to models that utilize pose guidance. When we combine the T2M model with the T2V model, the input is text, and the output is a video corresponding to that text. Consequently, in Table 1, we compared our approach with models that generate videos from text input. To evaluate our methodology, we needed text prompts that describe human actions. For this, we used slightly modified prompts from the test set of HumanML3D.

I hope this answers your question, and thank you again for your interest.
